content/docs/tasks/traffic-management/egress/egress-gateway/index.md

```diff
@@ -650,6 +650,58 @@ external service.
     sleep istio-proxy
     {{< /text >}}
 
+1.  Create the same destination rule as for the `sleep` pod in the `default` namespace to direct the traffic through the egress gateway:
```

Comment:
What is this for? This exact same destination rule is already created in step 1.

Reply:
No, it's created in a different namespace (`default` vs. `test-egress`).

Reply:
Doh I need to fix the command to add `-n test-egress` as I verified fixed the issue.